src/dvclive/dvc.py

```diff
@@ -71,25 +71,28 @@ def get_dvc_repo() -> Optional["Repo"]:
             return None
 
 
-def make_dvcyaml(live) -> None:
+def make_dvcyaml(live) -> None:  # noqa: C901
```

Comment:
Changes here are to change the paths to be relative to the new `dvc.yaml` location